pkgs/development/mobile/androidenv/tools.nix

```diff
@@ -6,7 +6,9 @@ deployAndroidPackage {
   nativeBuildInputs = [ makeWrapper ]
     ++ lib.optionals (os == "linux") [ autoPatchelfHook ];
   buildInputs = lib.optional (os == "linux") (
-      (with pkgs; [ glibc freetype fontconfig fontconfig.lib])
+      (with pkgs; [ glibc freetype fontconfig fontconfig.lib
+        stdenv.cc.cc.libgcc # fix for https://github.com/NixOS/nixpkgs/issues/226357
```

Comment:
```suggestion
        stdenv.cc.cc.libgcc or null # fix for https://github.com/NixOS/nixpkgs/issues/226357
```

Reply:
@github I'm in the Android app, and there's no apply button for the suggestion!

Anyway! I'll apply the suggestion as soon as possible, even though, I don't understand the reason behind the `or null` part. Would you explain more? In my view the `or null` is supposed to make it less likely to break in the future, right? But why do we want that when we can be explicit about the dependencies and have a fail-fast mindset to make reading, debugging, and fixing future problems easier?

Reply:
squashed and applied suggestion